integration_tests/dbt_project.yml

```diff
@@ -12,7 +12,12 @@ vars:
     linkedin_ads_creative_history_identifier: "linkedin_creative_history_data"
     linkedin_ads_ad_analytics_by_campaign_identifier: "linkedin_ad_analytics_by_campaign_data"
 
-    linkedin_ads_schema: linkedin_source_integration_tests_1
+    linkedin_ads_schema: linkedin_source_integration_tests_3
+
+models:
+  linkedin_source:
+    tmp:
+      +materialized: table
```

Comment:
Is this a necessary update?

Reply:
No thanks for catching. I was testing something out but should take it out now.